### ISSUE TITLE
Feature Request: Support --verbose option from xz

### ISSUE THREAD

**Issue by MadEgg** [vasi/pixz#57]
The xz command from xz-utils provides a option called --verbose that prints information on the progress, compression and estimated remaining time. When stdout is redirected, it will instead print a summary of the total compression when it's finished. It would be great if pixz could do the same.

**Comment by shmerl** [comment]
Yep, I just came here to request something similar. It's quite useful to see the progress.

**Comment by adamhotep** [comment]
The third-party [progress](http://github.com/Xfennec/progress) utility can't figure this command out either ☹️ (likely due to how the final file is assembled). Passing the input through [pv](https://www.ivarch.com/programs/pv.shtml) also doesn't work due to how the input is read (instantaneously in a single-file 200M test that got 50% compression). Therefore, a progress view must be internal to pixz.

**Comment by vasi** [comment]
I definitely use pv with pixz! It's only really useful when you have a multi-block file of course, as you noticed.

Adding built-in progress would be great though, PRs welcome.
